include/voxen/util/elapsed_timer.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <voxen/config.hpp>
+
+#include <fmt/format.h>
+
+namespace voxen
+{
+
+/**
+ * This is class is voxen-specific class for code segment measurements.
+ * The class will collect time point on constructor/stop method and print them into log (Debug level) automatically.
+ * Should be used like this:
+ *    ElapsedTimer test_timer("section name");
+ *    <measurable code here>
+ *    test_timer.stop()
+ */ 
+class ElapsedTimer 
+{
+public:
+    ElapsedTimer(std::string section_name, std::string format = "[Elapsed timer] execution of section \"{}\" tooks {} ms");
+    ElapsedTimer();
```

Comment:
Does default constructor have any sense with this class? I don't see an implementation for it either.